minecraft-plugin/src/main/java/com/pvchat/proximityvoicechat/plugin/ConfigManager.java

```diff
@@ -0,0 +1,56 @@
+package com.pvchat.proximityvoicechat.plugin;
+import org.bukkit.configuration.ConfigurationSection;
+import org.bukkit.configuration.file.FileConfiguration;
+import java.util.HashMap;
+import java.util.Set;
+
+public class ConfigManager {
+    ProximityVoiceChat pluginInstance;
+    private int maxHearDistance;
+    private int noAttenuationDistance;
+    private float linearAttenuationFactor;
+    private HashMap<String, String> playerLinks;
```

Comment:
IMO this should use player UUIDs. It should be safer than using names.

Reply:
Those are player UUID but stored as Strings.

Reply:
We could use UUID object instead